docs/integrations/jira.md

```diff
@@ -0,0 +1,6 @@
+# Jira
+Fairwinds Insights can connect to Jira in order to create Jira tickets
```

Comment:
Should we specify how to set this up?

Reply:
Hoping it's pretty straightforward if you follow the link here :)